tests/manage/z_cluster/test_rook_ceph_osd_flapping.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import pytest
+
+
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.ocs.cluster import ceph_health_check
+from ocs_ci.ocs.resources import pod
+from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    bugzilla,
+    skipif_ocs_version,
+    skipif_external_mode,
+    ignore_leftovers,
+)
+
+log = logging.getLogger(__name__)
+
+
+@brown_squad
+@tier2
+@ignore_leftovers
+@bugzilla("2223959")
+@skipif_external_mode
+@skipif_ocs_version("<4.14")
+@pytest.mark.polarion_id("OCS-XXXX")
+class TestRookCephOsdFlapping(ManageTest):
+    """
+    Test Rook Ceph OSD Flapping
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
```

Comment:
Need to add teardown check to validate whether osd is scaled up since the can test fail in between due to some issue.

Reply:
After reset the osd, we expected to get ceph ok state